### PR TITLE
ENG-8352 - Capture Package Name

### PR DESCRIPTION
**Pull request by kevinsites**
Updated to capture the bundleID `com.neuroid.sdksandbox.storyboard.dev`

Updated Event:
```
attrs=[
   versionName=2.3.1,
   versionNumber=3, 
   packageName=com.neuroid.sdksandbox.storyboard.dev, 
   applicationName=Neuro-ID iOS SDK Storyboard Sandbox
]
```

### CODE REVIEW

**Review by priya-xavier — APPROVED** [review]

**Review by kishisaka-nid — DISMISSED** [review]
pulled tested, 🚢

**Review by priya-xavier — APPROVED** [review]

**Review by kishisaka-nid — APPROVED** [review]

File: Podfile.lock [hunk 1]
```diff
@@ -25,8 +25,8 @@ SPEC CHECKSUMS:
   DSJSONSchemaValidation: 46e30ccfa9908c49d184f453d556f55aa0aac1ff
   FingerprintPro: 72e35dc0315b75ba36eafaaed996dbb2f35f7326
   JSONSchema: 52453ed2f570370b5e3ccd1c943afdfa5ea70db5
-  NeuroIDAdvancedDevice: 3974c46107a2e066e629e093e47ec40115f7a206
+  NeuroIDAdvancedDevice: 8c3da2679da4eca7cb499e111f7e725977be66bd
 
 PODFILE CHECKSUM: 090790257a48acef9caa8a475c7fb3aa58f17f20
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.14.3
```
Comment (kevinsites) [inline Podfile.lock:32]: This isn't what we normally want/do but we need to force the runner to get the latest ADV Lib

**Review by cometman — APPROVED** [review]
Looks average!

**Comment by cometman** [comment]
> Looks average!

JK, looks amazing :) 🚢

**Pull request merged**